index.html

```diff
@@ -4804,6 +5031,57 @@ <h4><dfn>Get Active Element</dfn></h4>
   <p>Otherwise, return <a>error</a> with <a>error code</a> <a>no such element</a>.
 </ol>
 </section> <!-- /Get Active Element -->
+
+<section>
+<h4><dfn>Get Element Shadow Root</dfn></h4>
+
+<table class="simple jsoncommand">
+  <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+  </tr>
+  <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/element/shadow</td>
```

Comment:
Shouldn't this include {element id}?

Reply:
Indeed it should. Thanks for catching that. Fixing that now.